### PR TITLE
attempt to improve readability of tpm2_takeownership

### PR DESCRIPTION
**Pull request by cardoe**
Attempted to improve the readability of tpm2_takeownership by replacing loops with function calls.

### CODE REVIEW

File: tools/tpm2_takeownership.c [hunk 1]
```diff
@@ -109,61 +122,50 @@ static bool change_hierarchy_auth(takeownership_ctx *ctx) {
...
+                ctx->is_hex_passwords, "Endorsement", TPM_RH_ENDORSEMENT))
+    {
+        return false;
+    }
+
+    // change lockout auth
+    if (!change_auth(ctx->sapi_context, &ctx->passwords.lockout,
+                ctx->is_hex_passwords, "Lockout", TPM_RH_LOCKOUT))
+    {
+        return false;
```
Comment (williamcroberts) [inline tools/tpm2_takeownership.c]: Style on this block is wrong, see: misc/coding_standard_c.txt

```
if ( condition ) {
  doit();
}
```

**Comment by williamcroberts** [comment]
You need to git pull --rebase origin master on your topic branch and push, so your changes have the .travis.yml changes that drop that git apply command.

**Comment by williamcroberts** [comment]
No updates, closing.

**Comment by cardoe** [comment]
Wow talk about being unfriendly towards contributors. I've been on travel for work and haven't had time. I've been active on other tickets and discussing things with @flihp.

**Comment by williamcroberts** [comment]
@cardoe I don't leave PRs open for long, it's clutter. Do you have the re-open button? When you have time just re-open it or submit a new PR.

**Comment by cardoe** [comment]
No I am unable to reopen it. Its been rebased. fwiw, I did rebase it after you asked but had a typo which caused it to fail. You can see from the timeline that the commit is AFTER you asking me to rebase which means that I did the rebase as you requested.

image

**Comment by cardoe** [comment]
So can you re-open this or do I need to make a new one?

**Comment by williamcroberts** [comment]
I cant, because it was force-pushed or recreated since close. https://github.com/01org/tpm2-tools/pull/412

**Pull request closed without merging**